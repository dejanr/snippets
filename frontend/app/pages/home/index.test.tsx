import { expect, test } from 'vitest'
import { render } from '@testing-library/react'
import { Home } from './index'

test('it renders', () => {
  const { container } = render(<Home />)

  expect(container.firstChild).toMatchSnapshot()
})
