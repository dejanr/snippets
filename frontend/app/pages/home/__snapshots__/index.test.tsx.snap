// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Home component snapshot 1`] = `
<div
  class="container mx-auto px-4 py-8 max-w-4xl"
>
  <main
    class="bg-white rounded-lg shadow-md p-6 my-8"
  >
    <h1
      class="text-3xl font-bold text-gray-800 mb-6"
    >
      Snippets
    </h1>
    <div
      class="space-y-4"
    >
      <p
        class="text-gray-600"
      >
        Welcome to code snippets.
      </p>
    </div>
  </main>
</div>
`;
